sagemaker-pyspark-sdk/src/sagemaker_pyspark/wrapper.py

```diff
@@ -157,3 +159,27 @@ def _to_java(self):
 
     def _from_java(self):
         raise NotImplementedError()
+
+
+# wrap scala.collection.immutable.List
+class ScalaList(SageMakerJavaWrapper):
+
+    def __init__(self, p_list):
+        self.p_list = p_list
+
+    def _to_java(self):
+        # Since py4j cannot deal with scala list directly
+        # we convert to scala listmap as an intermediate step
+        s_list = self._new_java_obj("scala.collection.immutable.ListMap")
+        key = 0
+        for elem in self.p_list:
+            tuple = self._new_java_obj("scala.Tuple2", key, elem)
+            s_list = getattr(s_list, "$plus")(tuple)
+            key += 1
+        s_list = getattr(s_list, "values")()
+        s_list = getattr(s_list, "toList")()
+
+        return s_list
+
+    def _from_java(self):
+        raise NotImplementedError()
```

Comment:
I think you are missing a new line at the end of the file.

Reply:
New line added. Seems my flake8 didn't work properly. Now it's fixed.